File: presence/concept-pl-execution-13-fixtures/concept_pulse.js
// concept: selective-retention
const signals = [2, 7, 1, 8, 2, 8];
const threshold = 5;
const scoreFloor = 100;
const retainedFloor = 3;
let score = 0;
let retained = 0;
for (let index = 0; index < signals.length; index += 1) {
  const value = signals[index];
  score += value * (index + 1);
  if (value >= threshold) retained += 1;
}
const verdict = score >= scoreFloor && retained >= retainedFloor ? "coherent" : "fragmented";
console.log(verdict + ":" + score + ":" + retained);

File: presence/concept-pl-execution-13-fixtures/concept_pulse.go
// concept: selective-retention
package main
import "fmt"
func main() {
  signals := []int{2, 7, 1, 8, 2, 8}
  threshold := 5
  scoreFloor := 100
  retainedFloor := 3
  score, retained := 0, 0
  for index, value := range signals {
    score += value * (index + 1)
    if value >= threshold { retained++ }
  }
  verdict := "fragmented"
  if score >= scoreFloor && retained >= retainedFloor { verdict = "coherent" }
  fmt.Printf("%s:%d:%d\n", verdict, score, retained)
}
